src/System.Management.Automation/engine/ComInterop/ComTypeClassDesc.cs

```diff
@@ -78,8 +68,8 @@ internal bool Implements(string itfName, bool isSourceItf)
         {
             if (isSourceItf)
```

Comment:
brackets?

Reply:
It's a copy of the .NET code. I prefer to stick to the style used in the code, so we can keep the diffs minimal when comparing with the .NET 5 code.

Reply:
Can you add a comment to that effect?  Chances are a code clean up effort will change it.

Reply:
Yes, I will add a comment in the `engine/ComInterop/README.md` about this.